Makefile

```diff
@@ -0,0 +1,17 @@
+.PHONY: i18n/sources.txt i18n/messages.pot
+
+i18n/sources.txt:
+	@grep \
+		--files-with-match \
+		--recursive \
+		--include \*.ts \
+		--extended-regexp "\.(gettext|dgettext|ngettext|dngettext|pgettext|dpgettext|npgettext|dnpgettext)\(" lib | sort > i18n/sources.txt
+
+i18n/messages.pot:
+	@xgettext \
```

Comment:
Is this tool cross-platform?

Reply:
I  believe the `Gettext` is cross-platform.

First of all  it worked on my Mac Book well. 

`GNU Gettext` should be worked even better on a Linux **distro**. 

For windows  user [GnuWin32][http://gnuwin32.sourceforge.net/packages/gettext.htm] should worked, but I have not a Window by hand, so I can not try it actually.

Reply:
Our goal is that from an env/devops point of view anyone can contribute to Renovate fairly easily whether using Linux, MacOS or Windows. So if this feature introduces any new feature, we should make sure it's not creating a burden which would discourage contributions. Right now I'm not sure we can satisfy that, so we need to think if/how these commands can be made more user friendly to run, including possibly by running the inside a Docker container. More Windows users are likely to have Docker installed than GnuWin32 for example

Reply:
As I mentioned, we should not let developers to execute any command related with `Gettext` on their own machines.
It's a repetitive and trublesome job to a developer. So I wrote a workflow to do that, please check my latest commits.
I  have also tested on my forked repo, please check the [PR](https://github.com/xingxing/renovate/pull/4).